frontend/packages/console-dynamic-plugin-sdk/src/app/redux.ts

```diff
@@ -1,5 +1,5 @@
-import coreReducer from './core/reducers/core';
+import { coreReducer } from './core/reducers/core';
 
-export const baseReducers = Object.freeze({
+export const SDKReducer = Object.freeze({
   core: coreReducer,
```

Comment:
I think we'll need another story to deal with the naming of this... the more I think of it the more we either need to provide some global prefix context for App stores (like [basename](https://v5.reactrouter.com/web/api/BrowserRouter/basename-string) prefixes but for Redux), or like just prefix everything with SDK; `sdkCore` for instance. 

Today, any app that uses `core` in their store will lose it when they spread us into theirs.

Reply:
yeah for now have made use of `sdkCore`.